gemd/entity/bounds/real_bounds.py

```diff
@@ -86,6 +86,62 @@ def contains(self, bounds: Union[BaseBounds, "BaseValue"]) -> bool:
 
         return bounds.lower_bound >= lower and bounds.upper_bound <= upper
 
+    def union(self, *others: Union["RealBounds", "ContinuousValue"]) -> "RealBounds":
+        """
+        Return the union of this bounds and other bounds.
+
+        The others list must also be Real Bounds or Values.
+
+        Parameters
+        ----------
+        others: Union[RealBounds, ContinuousValue]
+            Other bounds or value objects to include.
+
+        Returns
+        -------
+        RealBounds
+            The union of this bounds and the passed bounds
+
+        """
+        from gemd.entity.value.continuous_value import ContinuousValue
+
+        if any(not isinstance(x, (RealBounds, ContinuousValue)) for x in others):
+            misses = {type(x).__name__
+                      for x in others
+                      if not isinstance(x, (RealBounds, ContinuousValue))}
+            raise TypeError(f"union requires consistent typing; found {misses}")
+        lower = self.lower_bound
+        upper = self.upper_bound
+        unit_ = self.default_units
+        for bounds in others:
+            if isinstance(bounds, ContinuousValue):
+                this_lo, this_hi = bounds._to_bounds()._convert_bounds(unit_)
```

Comment:
`ContinuousValue` objects get converted to the default units, but if `others` contains other `RealBounds` objects, I don't see where those would be converted